docs/src/rsh/compute/index.md

```diff
@@ -1561,6 +1561,11 @@ where the first value is a `{!rsh} Fun([UInt], Bool)`
 which tells you if its argument is one of the enum's values,
 and the next N values are distinct `{!rsh} UInt`s.
 
+```reach
+load: ./examples/secured-loan/index.rsh
```

Comment:
`load` URL should only be prefixed with a '/' forward slash. '.' breaks the link in docs portal.